docs/public-networks/concepts/node-sync.md

```diff
@@ -97,6 +97,8 @@ to `false`. However, this will increase the sync time and disk space usage.
 
 Instead of downloading the [state trie](data-storage-formats.md) node by node, snap
 sync downloads as many leaves of the trie as possible, and reconstructs the trie locally.
```

Comment:
Just to clarify, when you say serving snap sync data, is that enabling the Besu node to act as a data provider for other nodes performing snap sync?